cves/CVE-2016-2327.yml

```diff
@@ -312,8 +329,10 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
```

Comment:
Would defense in depth apply here? If this caused a failure, applying defense in depth would allow the rest of the system to stay functional